Client/Application/AppDelegate.swift

```diff
@@ -117,7 +117,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
 
         browserViewController.restorationIdentifier = NSStringFromClass(BrowserViewController.self)
         browserViewController.restorationClass = AppDelegate.self
-        browserViewController.crashedLastSession = crashedLastSession
+        #if !MOZ_CHANNEL_FENNEC
```

Comment:
Add a comment about _why_ here. (e.g. simulator stop => crash, ads to tons of prompts on sim relaunches)